requirements/deploy.txt

```diff
@@ -2,8 +2,8 @@
 # http://initd.org/psycopg/docs/install.html#binary-install-from-pypi
 psycopg2==2.7.5 --no-binary psycopg2
 gunicorn==19.9.0
-pysolr==3.7.0
-django-redis-cache==1.7.1
+pysolr==3.8.1
```

Comment:
This looks like something used for hystack only https://github.com/django-haystack/pysolr, it was removed a time ago https://github.com/rtfd/readthedocs.org/pull/4039